README.md

```diff
@@ -176,6 +294,72 @@ Use `+` to separate the package names.
 curl https://npm.antfu.dev/versions/vite+vue+nuxt
 ```
 
+### `GET /engines/:pkg`
```

Comment:
Do we need this API in taze? If not I prefer to not having it for now to limit the surface of maintenance.